src/colmap/sfm/incremental_mapper.cc

```diff
@@ -686,6 +694,17 @@ bool IncrementalMapper::AdjustGlobalBundle(
                                              "registered for global "
                                              "bundle-adjustment";
 
+  BundleAdjustmentOptions ba_options_copy = ba_options;
```

Comment:
Maybe we should say ``ba_options_strict`` rather than ``ba_options_copy``.

Reply:
It's only strict if the condition below is true?

Reply:
Maybe ``ba_options_tmp``, ``ba_options_local``? Since we want to change this it s strange to name it ``copy`` to me. Anyway its just a name.

Reply:
Done.